content/v2.0-preview/_index.md

```diff
@@ -1,51 +1,30 @@
 ---
-title: "Overview"
+title: "Welcome"
 weight: -1
 cascade:
     version: "2.0-preview"
 ---
 
-{{< img src="/media/banner.png" alt="Crossplane Popsicle Truck" size="large" >}}
```

Comment:
I'm on the fence about whether we should keep this banner or not. LMK what you think.


<img width="1433" alt="Screenshot 2025-03-27 at 12 09 44 PM" src="https://github.com/user-attachments/assets/ce6f57f3-f059-466e-9743-a722b45c4076" />

vs

<img width="1415" alt="Screenshot 2025-03-27 at 12 10 03 PM" src="https://github.com/user-attachments/assets/9e954d6c-6a77-4d0c-8170-b2f2dca752c9" />

Reply:
I like having the visual element, but I feel it pushes the actual content too far down the page.

Reply:
i like having a crossplane banner, i love the artwork - can it maybe be made smaller? 🤔

Reply:
i like having a crossplane banner, i love the artwork - can it maybe be made smaller? 🤔

Reply:
Not sure, and I don't feel like there's time to investigate. I'm going to leave it out for now. I'm open to re-adding it in a future PR if we find some way to make it take up less of the page.